sonic-slave-bullseye/Dockerfile.j2

```diff
@@ -202,7 +202,7 @@ RUN apt-get update && apt-get install -y \
         libxml2-dev \
 # For BFN sdk build
         libusb-1.0-0-dev \
-        libcurl3-nss-dev \
+        libcurl4-openssl-dev \
```

Comment:
Can we change the one in sonic-slave-buster/Dockerfile.j2 as well? We are still using the docker image.

Reply:
Done